content/docs/command-reference/exp/run.md

```diff
@@ -90,8 +90,8 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
   [example](#example-run-a-grid-search)).
 
 - `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
-  default one will be generated otherwise, such as `exp-f80g4` (based on the
-  experiment's hash).
+  default one will be generated otherwise, such as `puffy-daks`. The name of the
+  experiment is exposed as an environment variable called `DVC_EXP_NAME`.
```

Comment:
Let's use a tip admon?

```suggestion
  default one will be generated otherwise, such as `puffy-daks`.
  
  <admon type="tip">
  
  The name of the experiment is exposed in env var `DVC_EXP_NAME`.
  
  </admon>
```